x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/transport/SecurityServerTransportInterceptor.java

```diff
@@ -148,7 +148,7 @@ private boolean shouldRequireExistingAuthentication() {
         try {
             sender.sendRequest(connection, action, request, options, handler);
         } catch (Exception e) {
-            handler.handleException(new TransportException("failed sending request", e));
+            handler.handleException(new SendRequestTransportException(connection.getNode(), "failed sending request", e));
```

Comment:
We should supply the action to the exception

```suggestion
            handler.handleException(new SendRequestTransportException(connection.getNode(), action, e));
```